eng/scripts/Yank-Crates.ps1

```diff
@@ -10,15 +10,17 @@ param(
 $ErrorActionPreference = 'Stop'
 Set-StrictMode -Version 2.0
 
+$env:CARGO_REGISTRY_TOKEN = $Token
```

Comment:
Should this even be passed in script? Why not just set it in `env` in the pipeline YAML? That's more typical anyway and has less risk of accidentally leaking it in output somehow.

Reply:
using `env:` from yaml now